Cargo.toml

```diff
@@ -24,14 +24,16 @@ cgmath = "0.16.0"
 collision = "0.18.0"
 error-chain = "0.11.0"
 fnv = "1.0.6"
+libc = "0.2.43"
+nav-types = "0.3"
 num = "0.1.36"
 num-traits = "0.1.36"
 pbr = "1.0.0-alpha.1"
 protobuf = "2.0.0"
+s2 = "0.0.8"
```

Comment:
Could it be a bit confusing to have a dependency of the same name as a module that uses it?

Reply:
Probably... how about `s2_geo` or `geo`?

Reply:
`s2_geo` sounds good

Reply:
Done, renamed it.